runusb

```diff
@@ -267,6 +268,23 @@ class RobotUSBHandler(USBHandler):
         else:
             LED_CONTROLLER.red()
 
+    def _rotate_old_logs(self, log_dir: str) -> None:
+        """
+        Add a suffix to the old log file, if it exists.
+
+        Suffixes are of the form log-<n>.txt, where <n> is the smallest
+        integer that didn't already exist.
+        """
+        if not os.path.exists(os.path.join(log_dir, 'log.txt')):
+            return
+        i = 1
+        while True:
```

Comment:
Suggestion: You can use `for i in itertools.count()` here to avoid needing to implement an infinite counter yourself.